utils/schema.js

```diff
@@ -13,7 +13,11 @@ export const UserSchema={
             then: Joi.required(),
             otherwise: Joi.optional()
         })
-    }),
+    }).unknown(true),
+    edit:Joi.object({
+        userName:Joi.string().min(3).max(100),
+        phone: Joi.string().min(7).max(11)
```

Comment:
I think we also need to check the email . but i will merge this pull request and i will fix it...